lblod/ember-rdfa-editor-lblod-plugins
# Migrate to pnpm

### Overview
This PR includes a migration from `npm` to `pnpm`, `pnpm` has the following advantages over `npm`:
- Faster dependency installation
- Option to use hard linking using the `file:` protocol: https://pnpm.io/cli/link#whats-the-difference-between-pnpm-link-and-using-the-file-protocol

##### connected issues and PRs:
N/A

### How to test/reproduce
- `pnpm install`
- `pnpm start` should still work as expected
- `pnpm prepack` should yield no errors
